### PR TITLE
Add Kubeedge file

### PR DESCRIPTION
**Pull request by odidev**
Added kubeedge.md file.

Signed-off-by: odidev [odidev@puresoftware.com](mailto:odidev@puresoftware.com)

### CODE REVIEW

File: content/sw_database/kubeedge.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    release_date: 15/05/2020
+
+
+optional_info:
+    homepage_url: https://kubeedge.io/en/
+    support_caveats:
+    alternative_options:
+    getting_started_resources:
+        arm_content:
+        partner_content:
```
Comment (odidev) [inline content/sw_database/kubeedge.md:18]: Check this: https://www.alibabacloud.com/blog/openyurt-the-practice-of-extending-native-kubernetes-to-the-edge_597903

File: content/sw_database/kubeedge.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    version_number: 1.3.0
+    release_date: 15/05/2020
+
+
+optional_info:
+    homepage_url: https://kubeedge.io/en/
+    support_caveats:
+    alternative_options:
+    getting_started_resources:
+        arm_content:
```
Comment (odidev) [inline content/sw_database/kubeedge.md:17]: Check this: https://community.arm.com/arm-community-blogs/b/tools-software-ides-blog/posts/q-a-with-priyanka-sharma-for-arm-devsummit-2020

File: content/sw_database/kubeedge.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+name: KubeEdge
+category: Automation & Configuration
+description: Kubernetes Native Edge Computing Framework
```
Comment (pruthviteja) [inline content/sw_database/kubeedge.md:4]: Maybe we can description as "KubeEdge is built upon Kubernetes and extends native containerized application orchestration and device management to hosts at the Edge."

File: content/sw_database/kubeedge.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        partner_content: https://www.alibabacloud.com/blog/openyurt-the-practice-of-extending-native-kubernetes-to-the-edge_597903
+        official_docs: https://kubeedge.io/docs/category/developer-guide/
+    arm_recommended_minimum_version:
+        version_number:
+        release_date:
+
+optional_hidden_info:
+    release_notes__supported_minimum: https://github.com/kubeedge/kubeedge/blob/master/CHANGELOG/CHANGELOG-1.3.md#v130
+    release_notes__recommended_minimum:
+    other_info: No ARM64 specific realease notes available but the first binary for ARM64 was released from v1.3.0.
```
Comment (odidev) [inline content/sw_database/kubeedge.md:27]: Please correct "realease" spelling and update sentence to "ARM64 support is not mentioned in any release notes, but the first binary for ARM64 was released for version v1.3.0."

**Pull request merged**